lib/errinj.h

```diff
@@ -0,0 +1,108 @@
+/**
+ *              Tempesta FW
+ *
+ * Error injection library.
+ *
+ * Copyright (C) 2015-2023 Tempesta Technologies, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License,
+ * or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59
+ * Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+ */
+#ifndef __LIB_ERRINJ_H__
+#define __LIB_ERRINJ_H__
+
+#include <linux/types.h>
+
+enum errinj_type {
+	/** boolean */
+	ERRINJ_BOOL,
+	/** int64_t */
+	ERRINJ_LONG,
+};
+
+enum errinj_state {
+	Errinj_I_S,
+	Errinj_Get_Val,
+	Errinj_Set_Val,
+	Errinj_I_F,
+};
+
+struct errinj {
+	/** Name, used to get/set errinj from userspace tests. */
+	const char *name;
+	/** Type, e.g. BOOL, UNT64, DOUBLE */
+	enum errinj_type type;
+	union {
+		/** bool parameter */
+		bool bparam;
+		/** integer parameter */
```

Comment:
Usually we write a single comment on top of a structure describing all the members - it's not good or bad, but it'd be good to adjust for code consistency

Reply:
Fixed